src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,55 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.ValidationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMAL_AGE = 18;
+    private static final int MINIMAL_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkUserDataNull(user);
+        checkLengthOfData(user);
+        checkIsUserAdult(user);
+        isLoginInStorage(user);
```

Comment:
these methods aren't placed in invocation order

I would expect that if `checkUserDataNull` is invoked first, then it should go first after the current method

it simplifies the reading process